esmvaltool/cmorizers/obs/cmor_config/GRACE.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Global attributes of NetCDF file
+attributes:
+  dataset_id: GRACE
+  project_id: OBS6
+  tier: 3
+  version: '1'
+  modeling_realm: satellite
+  source: ''
+  reference: 'grace'
+  comment: |
+    'Grace'
+
+# Variables to CMORize
+variables:
+  lweGrace:
+    mip: Lmon
+    raw: lwe_thickness
+    file: 'GRCTellus.JPL.200204_202003.GLO.RL06M.MSCNv02CRI.nc'
+
+auxfiles:
+  land_mask:
+    'LAND_MASK.CRI.nc'
+  scale_factor:
+    'CLM4.SCALE_FACTOR.JPL.MSCNv02CRI.nc'
+
+grace_table:
+  'https://podaac-tools.jpl.nasa.gov/drive/files/allData/tellus/L3/docs/GRACE_GRACE-FO_Months_RL06.csv'
```

Comment:
Wouldn't it be better to use the name of the csv file instead of its url? Otherwise, the csv file name is hardcoded in the cmorizer